charts/rancher-logging/rancher-logging/100.0.0+up3.12.0/templates/loggings/k3s/configmap.yaml

```diff
@@ -18,7 +18,7 @@ data:
     [INPUT]
         Name              systemd
         Tag               k3s
-        Path              {{ .Values.systemdLogPath | default "/var/log/journal" }}
+        Path              {{ .Values.systemdLogPath }}
```

Comment:
Is this a required path in the UI ? If not should it be ?

Reply:
There is an issue to add it to the UI but we decided it is a v2.6.1 issue.

Reply:
I should mention there is still a default in values.yaml of `/var/log/journal`